tests/integration/test_outcome.py

```diff
@@ -245,7 +246,10 @@ def test_outcome_source(relay, mini_sentry):
 
 @pytest.mark.parametrize("num_intermediate_relays", [1, 3])
 def test_outcome_forwarding(
-    relay, relay_with_processing, outcomes_consumer, num_intermediate_relays,
+    relay,
+    relay_with_processing,
+    outcomes_consumer,
+    num_intermediate_relays,
```

Comment:
From "CI / Lint":
```
Run black --check py tests --exclude '\.eggs|sentry_relay/_lowlevel.*'
would reformat /home/runner/work/relay/relay/tests/integration/test_outcome.py
Oh no! 💥 💔 💥
```

I think there must be a version/config mismatch with Black, because my local Git hooks insisted I change it *to* this before committing.